### ISSUE TITLE
Accessibility investigation.

### ISSUE THREAD

**Issue by etoledom** [wordpress-mobile/gutenberg-mobile#476]
At some point it will be necessary to investigate how to apply the native Accessibility in React-Native.

There is an articule about this here: https://facebook.github.io/react-native/docs/accessibility.html

With that investigation, we should be able to figure out what is the work we need to do to make Gutenberg Mobile accessible, and create a guideline for all future UI development work to comply to.

Related issues:
https://github.com/wordpress-mobile/gutenberg-mobile/issues/14
https://github.com/wordpress-mobile/gutenberg-mobile/issues/598
https://github.com/wordpress-mobile/gutenberg-mobile/issues/605
https://github.com/wordpress-mobile/gutenberg-mobile/issues/697

**Actions**, improve accessibility on:
- [x] Toolbar: PR #793
- [x] Image block (deselected): PR #794 
- [x] Image block (selected + toolbar): PR #920
- [x] Paragraph block: PR #923

**Comment by rachelmcr** [comment]
Do we have a timeline or estimate for when we'll work on this? I started looking at accessibility in the WPiOS integration and ran into major accessibility issues (no labels for toolbar buttons, block-level buttons completely inaccessible). That's a concern for our users and a blocker for automating tests that use Gutenberg.

**Comment by etoledom** [comment]
We will start working on this next spring 👍

**Comment by rachelmcr** [comment]
Excellent, thanks for the update! @wordpress-mobile/platform-9 also agreed that we're happy to help with adding accessibility labels and identifiers as needed (especially for test automation but also as we come across issues). So once there's a documented/agreed-upon process for making Gutenberg accessible please let us know and we can contribute to any work that's needed in the existing UI!

**Comment by etoledom** [comment]
Closing since the main issues have been solved.
A few [new independent tickets](https://github.com/wordpress-mobile/gutenberg-mobile/issues?utf8=✓&q=is%3Aopen+label%3AAccessibility+is%3Aissue) have been opened too.
